src/components/theme/color-mode/default-color-mode.ts

```diff
@@ -46,6 +48,7 @@ export const DEFAULT_COLOR_MODE: IColorsType = {
   input: BASE_COLORS.gray700,
   inputBorder: BASE_COLORS.gray400,
   placeholder: BASE_COLORS.gray600,
+  borderColor: BASE_COLORS.gray300,
```

Comment:
This color variable has been declared which is `border`.

Reply:
duplicated variable，delete borderColor，use  `border`.